processes/release_processes/scripts/start_publish_cromwell.sh

```diff
@@ -0,0 +1,39 @@
+# Launch a Cromwell server suited for publishing new versions of Cromwell. This uses the Local backend
+# and presumes the existence of an appropriately configured MySQL instance. See
+# `start_publish_mysql_docker.sh` for launching a containerized version of such a MySQL instance.
+#
+# Vault authentication is required to render config files, see
+# https://github.com/broadinstitute/dsde-toolbox#authenticating-to-vault.
+# Once authenticated, the Cromwell config files can be rendered with:
+#
+# sbt renderCiResources
```

Comment:
Did you consider running this as part of the script?

Reply:
ooh good idea